crates/era-utils/src/history.rs

```diff
@@ -23,9 +23,9 @@ use tracing::info;
 ///
 /// Returns current block height.
 pub fn import<Downloader, Era, P, B, BB, BH>(
-    mut downloader: Downloader,
+    downloader: &mut Downloader,
```

Comment:
I see,

could we avoid this if we create the stream on demand in the EraStage,

so maybe EraStage either requires that the Downloader type there is Clone or takes something that can create a new Downloader on demand?

like a boxed closure for example that returns `BoxStream<eyre::Result<Era>` for example, which could be something like

```rust
 Box::new( move || read_dir(path).boxed())
Box::new(move || EraStream::new(client.clone(), config.clone()).boxed())
```

for example

so that we have

```rust
EraStage {
  downloader: Box<dyn Fn() -> ..>
}
```

maybe a helper trait would this look a bit nicer